langchain4j-mistral-ai/src/test/java/dev/langchain4j/model/mistralai/MistralAiChatModelIT.java

```diff
@@ -457,6 +475,94 @@ void should_execute_multiple_tools_using_model_open8x22B_then_answer() {
         AiMessage aiMessage2 = response2.content();
         assertThat(aiMessage2.text()).contains("T123");
         assertThat(aiMessage2.text()).containsIgnoringCase("paid");
+      
+        assertThat(Arrays.asList("March 11, 2024","2024-03-11"))
+                .anySatisfy(date -> assertThat(aiMessage2.text())
+                        .containsIgnoringWhitespaces(date));
+        assertThat(aiMessage2.toolExecutionRequests()).isNull();
+
+        TokenUsage tokenUsage2 = response2.tokenUsage();
+        assertThat(tokenUsage2.inputTokenCount()).isGreaterThan(0);
+        assertThat(tokenUsage2.outputTokenCount()).isGreaterThan(0);
+        assertThat(tokenUsage2.totalTokenCount())
+                .isEqualTo(tokenUsage2.inputTokenCount() + tokenUsage2.outputTokenCount());
+
+        assertThat(response2.finishReason()).isEqualTo(STOP);
+    }
+
+    @Test
+    void should_code_generation_using_model_openCodestralMamba_and_return_finishReason(){
+        // given
+        UserMessage userMessage = userMessage("Write a java code for fibonacci");
+
+        // when
+        Response<AiMessage> response = openCodestralMamba.generate(userMessage);
+        System.out.println(response.content().text());
+
+        // then
+        assertThat(response.content().text()).isNotBlank();
+
+        TokenUsage tokenUsage = response.tokenUsage();
+        assertThat(tokenUsage.inputTokenCount()).isGreaterThan(0);
+        assertThat(tokenUsage.outputTokenCount()).isGreaterThan(0);
+        assertThat(tokenUsage.totalTokenCount())
+                .isEqualTo(tokenUsage.inputTokenCount() + tokenUsage.outputTokenCount());
+
+        assertThat(response.finishReason()).isEqualTo(STOP);
+    }
+
+    @Test
+    void should_execute_multiple_tools_using_model_openMistralNemo_then_answer() {
```

Comment:
same here. It does not seem to differe from `should_execute_multiple_tools_using_model_open8x22B_then_answer`